1-org/envs/shared/org_policy.tf

```diff
@@ -177,3 +177,66 @@ resource "google_access_context_manager_access_policy" "access_policy" {
   parent = "organizations/${var.org_id}"
   title  = "default policy"
 }
+
+
+module "restrict_procotol_fowarding" {
+  source          = "terraform-google-modules/org-policy/google"
+  version         = "~> 5.1"
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "list"
+  allow           = ["internal"]
+  enforce         = "true"
+  constraint      = "constraints/compute.restrictProtocolForwardingCreationForTypes"
+}
+
+module "disable_vpc_external_ipv6" {
+  source          = "terraform-google-modules/org-policy/google"
+  version         = "~> 5.1"
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "boolean"
+  enforce         = "true"
+  constraint      = "constraints/compute.disableVpcExternalIpv6"
+}
+
+
+module "disable_service_account_key_upload" {
+  source          = "terraform-google-modules/org-policy/google"
+  version         = "~> 5.1"
+  organization_id = local.organization_id
+  folder_id       = local.folder_id
+  policy_for      = local.policy_for
+  policy_type     = "boolean"
+  enforce         = "true"
+  constraint      = "constraints/iam.disableServiceAccountKeyUpload"
+}
+
+data "google_organization" "org" {
+  organization = var.org_id
```

Comment:
This should not use the `var.org_id`, 
the user must provide a list of domains, like it is done in the domain restricted sharing. 
https://github.com/terraform-google-modules/terraform-google-org-policy/blob/master/modules/domain_restricted_sharing/main.tf#L17-L20

and it must be a different list of the list used for domain restricted sharing